RELEASE_NOTES.md

```diff
@@ -9,3 +9,17 @@ python -m pip install frequenz-sdk
 ```
 
 The GitHub issues were also improved, adding templates for [reporting issues](https://github.com/frequenz-floss/frequenz-sdk-python/issues/new?assignees=&labels=priority%3A%E2%9D%93%2C+type%3Abug&template=bug.yml) and [requesting features](https://github.com/frequenz-floss/frequenz-sdk-python/issues/new?assignees=&labels=part%3A%E2%9D%93%2C+priority%3A%E2%9D%93%2C+type%3Aenhancement&template=feature.yml). Users are also pointed to the [Discussion forums](https://github.com/frequenz-floss/frequenz-sdk-python/issues/new/choose) when trying to open an issue if they have questions instead. Also many labels are assigned automatically on issue and pull request creation.
+
+## Upgrading
+
+Even if there are no breaking changes, if when upgrading you see this error in
+your local environment:
```

Comment:
> Even if there are no breaking changes, if when upgrading you see this error in
your local environment.

Even if there are no breaking changes, if you see this error in
your local environment while upgrading:

Reply:
Yup, it sounds weird. I will fix.